app/views/issues/_comments.html.erb

```diff
@@ -0,0 +1,2 @@
+<h3>Comments <span class="badge"><%= @issue.comments.count %></span></h3>
```

Comment:
This whole partial seems unnecessary, I'd move the two lines to Issues#show or if you really want to extract it, I'd move it under `./app/views/comments` merging the <h3> into the `comments/feed` partial. That way you can re-use it for Evidence, Notes, etc.